src/test_lib_json/jsontest.cpp

```diff
@@ -275,13 +275,12 @@ bool Runner::runAllTest(bool printSummary) const {
     }
     return true;
   } else {
-    for (unsigned int index = 0; index < failures.size(); ++index) {
-      TestResult& result = failures[index];
+    for (auto & result : failures) {
       result.printFailure(count > 1);
     }
 
     if (printSummary) {
-      unsigned int failedCount = static_cast<unsigned int>(failures.size());
+      auto failedCount = static_cast<unsigned int>(failures.size());
```

Comment:
I don't see much reason not to just stick with `size_t` for these counts and format them with`%zu`.